### PR TITLE
Add cronsetup package to interface go-etcd-cron with scalingo's services

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: cronsetup/cronsetup.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	"fmt"
+
+	"github.com/Scalingo/go-etcd-cron"
+	"github.com/Scalingo/go-utils/logger"
+	"github.com/coreos/etcd/clientv3"
+)
+
+type SetupOpts struct {
+	EtcdConfig func() (clientv3.Config, error)
+	Jobs       []etcdcron.Job
```
Comment (EtienneM) [inline cronsetup/cronsetup.go]: Shouldn't you import `etcdcron`?

File: cronsetup/cronsetup.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	if err != nil {
+		return nil, fmt.Errorf("fail to get etcd v3 config: %v", err)
+	}
+
+	etcdMutexBuilder, err := etcdcron.NewEtcdMutexBuilder(etcdConfig)
+	if err != nil {
+		return nil, fmt.Errorf("fail to get etcd mutex builder: %v", err)
+	}
+
+	funcCtx := func(ctx context.Context, j etcdcron.Job) context.Context {
```
Comment (EtienneM) [inline cronsetup/cronsetup.go]: Is it really what you wanted to do? A function which just do logging?

File: cronsetup/cronsetup.go (hunk 1 above)
Comment (Soulou) [reply cronsetup/cronsetup.go]: the package is imported by `+	"github.com/Scalingo/go-etcd-cron"`

File: cronsetup/cronsetup.go (hunk 2 above)
Comment (Soulou) [reply cronsetup/cronsetup.go]: What else would you like to do?

File: cronsetup/cronsetup.go (hunk 1 above)
Comment (EtienneM) [reply cronsetup/cronsetup.go]: Shouldn't you import it with `etcdcron "github.com/Scalingo/go-etcd-cron"` so that it works?

**Pull request merged**